src/components/Avatar/Avatar.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+    import type { Snippet } from 'svelte';
+    import type { Action } from 'svelte/action';
+
+    interface AvatarProps {
+        href?: string;
+        action?: Action<HTMLElement, string>;
+        actionParams?: string;
+
+        // Root (figure)
+        base?: string;
+        background?: string;
+        text?: string;
+        fontWeight?: string;
```

Comment:
So I have mixed feelings about `weight` here. The Tailwind convention is: `font-bold`. I think in most cases we should try to stick with the first word before the dash, so this would just be `font`. However, both Font Family and Font Weight use `font-`, which is a bit unique in Tailwind. Technically there's not a lot of downside to having `font` serve for BOTH family and weight. Props can support multiple classes like that. Thoughts?

Reply:
I agree, `font` is easier to use and can serve both purposes.